<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindowClass</class>
 <widget class="QMainWindow" name="ChatWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>383</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="wdtInfo" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="wdtView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wdtToolBar" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="wdtEdit" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
